models/webhook.go

```diff
@@ -171,8 +171,8 @@ func NewWebhookEventCaseCommentCreated(c Case) WebhookEventContent {
 	return newWebhookContentCase(WebhookEventType_CaseCommentCreated, c.Id)
 }
 
-func NewWebhookEventCaseFileCreated(c Case) WebhookEventContent {
-	return newWebhookContentCase(WebhookEventType_CaseFileCreated, c.Id)
+func NewWebhookEventCaseFileCreated(caseId string) WebhookEventContent {
+	return newWebhookContentCase(WebhookEventType_CaseFileCreated, caseId)
```

Comment:
Should we do a breaking change to dispatch "multi file uploads" (instead of one event per file)

Reply:
We keep the same event, I'll just update the doc on readdme